c7n/resources/sqs.py

```diff
@@ -160,42 +161,101 @@ class RemovePolicyStatement(RemovePolicyBase):
                     statement_ids: matched
     """
 
-    permissions = ('sqs:GetQueueAttributes', 'sqs:RemovePermission')
+    permissions = ("sqs:GetQueueAttributes", "sqs:RemovePermission")
 
     def process(self, resources):
         results = []
-        client = local_session(self.manager.session_factory).client('sqs')
+        client = local_session(self.manager.session_factory).client("sqs")
         for r in resources:
             try:
                 results += filter(None, [self.process_resource(client, r)])
             except Exception:
-                self.log.exception(
-                    "Error processing sqs:%s", r['QueueUrl'])
+                self.log.exception("Error processing sqs:%s", r["QueueUrl"])
         return results
 
     def process_resource(self, client, resource):
-        p = resource.get('Policy')
+        p = resource.get("Policy")
         if p is None:
             return
 
-        p = json.loads(resource['Policy'])
+        p = json.loads(resource["Policy"])
         statements, found = self.process_policy(
-            p, resource, CrossAccountAccessFilter.annotation_key)
+            p, resource, CrossAccountAccessFilter.annotation_key
+        )
 
         if not found:
             return
 
         for f in found:
-            client.remove_permission(
-                QueueUrl=resource['QueueUrl'],
-                Label=f['Sid'])
+            client.remove_permission(QueueUrl=resource["QueueUrl"], Label=f["Sid"])
 
-        return {'Name': resource['QueueUrl'],
-                'State': 'PolicyRemoved',
-                'Statements': found}
+        return {
+            "Name": resource["QueueUrl"],
+            "State": "PolicyRemoved",
+            "Statements": found,
+        }
 
 
-@SQS.action_registry.register('delete')
+@SQS.action_registry.register("add-statements")
+class AddPolicyStatements(AddPolicyBase):
```

Comment:
can you revert the style changes minus the add, there's a lot of unrelated line noise else on this pr